src/documents/RemoteModelAPI.md

```diff
@@ -0,0 +1,316 @@
+---
+title: RemoteModel API
+layout: default
+category: Infusion
+---
+
+`fluid.remoteModelComponent` builds on top of [`fluid.modelComponent`](ComponentGrades.md) with the purpose of providing a buffer between a local and
+remote model that are attempting to stay in sync. For example a local model is being updated by user interaction, this is sent back
+to a remote server, which in turn tries to update the local model. If additional user actions occur during the roundtrip, an
+infinite loop of updates may occur. `fluid.remoteModelComponent` solves this by restricting reading and writing to a single request at a time,
```

Comment:
Should say it actually does a little more than this, in that it will make some attempt to "rebase" local changes that have occurred with respect to remote ones.